### DISCUSSION TITLE
Downloading C/C++ extension error

### DISCUSSION THREAD

**Discussion by vaibhavbunny** [microsoft/vscode-cpptools#10612] [Q&A]
Error while installing 'C/C++' extension. Please check the [log](command:workbench.action.showWindowLog) for more details.

this is the error I am facing while installing the c/c++ extension ,what should I do now.

**Comment by sean-mcmanus** [comment]
Can you uninstall and try again or try downloading and installing the vsix (see instructions at https://github.com/microsoft/vscode-cpptools/releases/tag/v1.14.3)? Our extension isn't responsible for installing itself so you should file an issue at https://github.com/microsoft/vscode/issues .
